Dockerfile.builder

```diff
@@ -1,4 +1,4 @@
-FROM golang:1.19
+FROM golang:1.20
```

Comment:
arent we moving to `1.20.11` ??

Reply:
I was using generic release following old pattern. If we want specific version updating.